CONTRIBUTING.md

```diff
@@ -1,5 +1,5 @@
 # Contributing to sbi
 
-The latest contributing guide is available in the repository at `docs/contribute.md`, or online at:
+The latest contributing guide is available in the repository at `mkdocs/docs/contribute.md`, or online at:
```

Comment:
I think this actually correct (we have both `docs` and `mkdocs` folders at the moment). 

But I suggest changing this to an actual link: 
```suggestion
The latest contributing guide is available in the repository at [`docs/contribute.md`](docs/contribute.md), or online at:
```

Reply:
Yeah, true.